templates/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,47 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.192.0/containers/php
+{
+	"name": "PHP",
+	"build": {
+		"dockerfile": "Dockerfile",
+		"args": {
+			"PHP_VERSION": "8",
```

Comment:
This doesn't match the format of the "8.0" default in the DockerFile, is the format ok as just "8"?

Reply:
Just seen this after merge, sorry. So yes, you can still use 8, but it does make sense to make 8.0 the default here as well for consistency. Will open a new PR for it. Good shout!